Makefile

```
@@ -0,0 +1,31 @@
+.PHONY: deploy-prod deploy-server-send deploy-server-swap deploy-scheduler deploy-worker deploy-tx-indexer deploy-grafana
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)